.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,108 @@
+---
+# TL;DR: this workflow runs the benchmarks on the latest commit on the main
+# and then deploys the updated benchmarks to
+# `nilearn.github.io/benchmarks <https://nilearn.github.io/benchmarks/>`_.
+# You can trigger it manually by adding ``[bm]`` to the commit message on the
+# main. It will also run on a schedule every day at 10pm UTC.
```

Comment:
Every day ? sounds a bit heavy.

Reply:
Strong +1. I think @Remi-Gau [made the same point](https://github.com/nilearn/nilearn/issues/4573#issuecomment-2750913464) in the linked issue.

Reply:
Ok let's do weekly then.